frontend/src/core/i18n/en.json

```
@@ -7,6 +7,7 @@
     "settings": "Settings"
   },
   "settings": {
+    "actions": { "delete": "Delete", "cancel": "Cancel" },
```

Biome (2.4.15)

[error] The key actions was already declared.

(lint/suspicious/noDuplicateObjectKeys)